trojsten/utils/templatetags/progressbar.py

```diff
@@ -0,0 +1,57 @@
+from django.utils import timezone
+from django.utils.translation import ngettext
+from django.conf import settings
+from django import template
+
+
+register = template.Library()
+
+
+def get_progressbar_data(start, end):
```

Comment:
Len taka drobnost, toto nie je template tag, tak by to asi malo byt len v utils.progressbar